{"uuid": "54c33661-cfe0-4a33-ba87-0fb827a6a7cd", "children": ["9e83c82a-a33a-43e7-a3b7-2dbfa92365ea", "2e499a2e-bf82-462d-80d6-6aff8b4ce1ca", "fa2bd223-b3d2-415b-becb-74e0ad780cf2", "33987fb7-4a91-4114-82c9-cbb2154e9822"], "befores": [{"name": "setup", "status": "passed", "start": 1693482071636, "stop": 1693482103363}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=53524): Max retries exceeded with url: /session/e2149918bed0360ece8e089fee5cd01c/window (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10c821590>: Failed to establish a new connection: [Errno 61] Connection refused'))\n", "trace": "  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/tests/vwo_login/conftest.py\", line 15, in setup\n    driver.close()\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 456, in close\n    self.execute(Command.CLOSE)\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 343, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 291, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 312, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/_request_methods.py\", line 110, in request\n    return self.request_encode_url(\n           ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/_request_methods.py\", line 143, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/poolmanager.py\", line 443, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 844, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"/Users/megha/PycharmProject/pywebautomationframework/venv/lib/python3.11/site-packages/urllib3/util/retry.py\", line 515, in increment\n    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1693482130210, "stop": 1693482130324}], "start": 1693482071636, "stop": 1693482130324}